Training5/src/task-2/src/pages/my-info/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useEffect} from 'react';
+import {useDispatch} from "react-redux";
+import axios from "axios";
+import {base_url} from "../../App";
+import {getUserDetail} from "../../reducers/auth.slice";
+import {useAppSelector} from "../../hook/useAppSelector";
+
+const MyInfo = () => {
+    const dispatch = useDispatch();
+    const userData = useAppSelector((state) => state.root.auth.data)
```

Comment:
we should not get a token in every component that we call to the api, please review the Axios instance and get rid of duplication